webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -90,6 +95,22 @@ const TaskActionDropdown: React.FC<Props> = ({ task, onComplete }: Props) => {
           if (!isExperiment) break;
           await unarchiveExperiment({ experimentId: id });
           if (onComplete) onComplete(action);
+          break;
+        case Action.Delete:
+          if (!isExperiment) break;
+          Modal.confirm({
```

Comment:
you could make this modal a component since it looks like you use it twice, here and in ExperimentDetailsHeader

Reply:
I thought about that but then in order to make it a bit more useful, I would have needed to expose basically the same options Modal.confirm takes so I went back to using it directly. We're gonna be using confirm modals more but in order to be able to use it we'd have to expose most of these options. wdy think?

Reply:
Yeah, that's true. I think as long as the modals stay small like this it's fine, and just extract them into a component if they start getting their own logic.